### DISCUSSION TITLE
Binary compatibility on import of a vb6 COM dll

### DISCUSSION THREAD

**Discussion by dude202** [twinbasic/twinbasic#1904] [Q&A] [answered]
As far as I can tell, twinbasic does not offer an option to make its dll binary compatible to the imported vb6 dll. Do you plan to offer that option (it would be just an additional checkbox on the import dialog). I have a LOT of classes and interfaces, no way I could do it manually after the import. The option is a must if one is to have twinbasic dll as a drop-in replacement for vb6 dll. And a showstopper otherwise.

**Comment by WaynePhillipsEA** [comment, marked as answer]
Correct, twinBASIC doesn't currently support this.

It is planned, yes.
